File: PASCAL/Cap02/c02ex01.pas
program c02ex01;

uses
  Math;

begin
  // Operadores binários

  WriteLn(5 + 3);       // Saída: 8
  WriteLn(10 - 4);      // Saída: 6
  WriteLn(2 * 6);       // Saída: 12
  WriteLn(10 / 3);      // Saída: 3.3333... (quociente real)
  WriteLn(10 div 3);    // Saída: 3 (quociente inteiro)
  WriteLn(10 mod 3);    // Saída: 1 (resto da divisão)

  // Operadores unários

  WriteLn(+5);          // Saída: 5 (mantém positivo)
  WriteLn(-8);          // Saída: -8 (inverte sinal)

  // Exponenciação

  WriteLn(Power(2, 3)); // Saída: 8 (2 elevado a 3)

end.

File: CPP/Cap02/c02ex01.cpp
// c02ex01.cpp

#include <iostream>
#include <cmath>

using namespace std;

int main(void) {
  // Operadores binários
  cout << 5 + 3 << endl;     // Saída: 8
  cout << 10 - 4 << endl;    // Saída: 6
  cout << 2 * 6 << endl;     // Saída: 12
  cout << 10.0 / 3 << endl;  // Saída: 3.3333 (quociente real)
  cout << 10 / 3 << endl;    // Saída: 3 (quociente inteiro)
  cout << 10 % 3 << endl;    // Saída: 1 (resto da divisão)

  // Operadores unários
  cout << +5 << endl;        // Saída: 5 (mantém positivo)
  cout << -8 << endl;        // Saída: -8 (inverte sinal)

  // Exponenciação
  cout << pow(2, 3) << endl; // Saída: 8 (2 elevado a 3)

  return 0;
}
